src/app-layer-htp-range.h

```diff
@@ -55,18 +55,18 @@ typedef struct HttpRangeContainerFile {
     uint8_t *key;
     /** key length */
     uint32_t len;
-    /** pointer to hashtable data, for use count */
-    THashData *hdata;
     /** expire time in epoch */
     uint32_t expire;
+    /** pointer to hashtable data, for use count */
+    THashData *hdata;
     /** total epxected size of the file in ranges */
     uint64_t totalsize;
-    /** file flags */
-    uint16_t flags;
     /** file container, with only one file */
     FileContainer *files;
     /** red and black tree list of ranges which came out of order */
     struct HTTP_RANGES fragment_tree;
+    /** file flags */
+    uint16_t flags;
```

Comment:
Is there a tool for this that we can use in CI ?

Reply:
There is `pahole` that I use, at least for more complex layouts. Its a bit fragile though. Not sure if using it in CI would work.